install_sct

```diff
@@ -41,9 +41,27 @@ DATA_DIR="data"
 PYTHON_DIR="python"
 BIN_DIR="bin"
 
+function die {
+    echo -e "\n\033[0;31m$1\033[0m\n"
```

Comment:
in the rest of this script, the convention is two chars for tabulation, so i would stick to 2 chars everywhere